### PR TITLE
Feat/#1 database

### PR DESCRIPTION
**Pull request by jordilaforge**

### CODE REVIEW

File: src/handlers/booking.handlers.test.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import {GetAllBookingHandler} from "./booking.handlers"
+import {Booking} from "../models/Booking"
+
+describe("booking.handlers", () => {
+  describe("GetBookingHandler", () => {
+    test("Should have a name and version of the package.json", async done => {
+      const req: any = {}
```
Comment (hirsch88) [inline src/handlers/booking.handlers.test.ts]: Could your move the req, res and container object to shared `mocks` folder. I think we will use them in almost every handler test.

File: src/handlers/booking.handlers.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {RequestHandler, Response} from "express"
+import {Log} from "../lib/log"
+import {Container} from "../container"
+import {Booking} from "../models/Booking"
+
+export const GetAllBookingHandler = (
+  {repositories}: Container,
+): RequestHandler => async (req, res): Promise<Response> => {
+  Log.info("GetAllBookingHandler")
```
Comment (hirsch88) [inline src/handlers/booking.handlers.ts]: Could you add some better log messages :-)

File: src/handlers/booking.handlers.test.ts (hunk 1 above)
Comment (jordilaforge) [reply src/handlers/booking.handlers.test.ts]: ok i try

File: src/handlers/booking.handlers.ts (hunk 2 above)
Comment (jordilaforge) [reply src/handlers/booking.handlers.ts]: sure

**Pull request merged**